### PR TITLE
WWW-403 Setting border-radius to 0 instead of none.

### PR DESCRIPTION
**Pull request by holingpoon**
I think this is the line that is causing problems. Please review. Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ### CHANGE LOG
 
+### v.2.4.10
```
Comment (holingpoon) [inline CHANGELOG.md]: @ktp242 Like this one for CHANGELOG?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+> Fixes outstanding HTML conformance issues
+  - Fixes CSS: border-radius: Too many values or values are not recognized
+
```
Comment (holingpoon) [inline CHANGELOG.md]: @ktp242 Like this one for CHANGELOG?

Comment (ktp242) [reply CHANGELOG.md]: I think you can add the change to the already existing v2.4.10

**Comment by holingpoon** [comment]
Merged into branch v2.4.11, will submit another pull request

**Pull request closed without merging**